src/components/Header/NetworkSelector.tsx

```diff
@@ -292,9 +293,6 @@ export default function NetworkSelector() {
         if (!skipToggle) {
           toggle()
         }
-        history.replace({
-          search: replaceURLParam(history.location.search, 'chain', getChainNameFromId(targetChain)),
-        })
       } catch (error) {
         console.error('Failed to switch networks', error)
 
```

Comment:
the logic here to revert url param i'd like to move as part of the hook instead, by returning success/failure from this async function. later!